commands/command_prune.go

```diff
@@ -176,28 +177,29 @@ func prune(fetchPruneConfig lfs.FetchPruneConfig, verifyRemote, dryRun, verbose
 	}
 
 	if len(prunableObjects) == 0 {
-		Print("Nothing to prune")
 		return
 	}
+
+	info := tasklog.NewSimpleTask()
+	logger.Enqueue(info)
 	if dryRun {
-		Print("%d files would be pruned (%v)", len(prunableObjects), humanize.FormatBytes(uint64(totalSize)))
+		info.Logf("prune: %d file(s) would be pruned (%s)", len(prunableObjects), humanize.FormatBytes(uint64(totalSize)))
 		if verbose {
-			Print(verboseOutput.String())
+			for _, item := range verboseOutput {
+				info.Logf("\n * %s", item)
+			}
 		}
+		info.Complete()
 	} else {
-		Print("Pruning %d files, (%v)", len(prunableObjects), humanize.FormatBytes(uint64(totalSize)))
 		if verbose {
-			Print(verboseOutput.String())
+			for _, item := range verboseOutput {
+				info.Logf("\n%s", item)
+			}
 		}
+		info.Complete()
```

Comment:
This could be moved to a `defer`, allowing the one on L192 to be removed too. I point it out because I think having a consistent pattern for dealing with loggers and tasks is important. I really like how the `tasklog` package is shaping up :)

Reply:
Hm, I don't think that these calls to `Complete()` can be deferred, since `pruneDeleteFiles` creates a new task, which (for the `*tasklog.Logger` to output), requires there to be no actively running task.